include/Settings/SpriteSheet.hpp

```diff
@@ -152,6 +152,8 @@ class SpriteSheet final : public Core::Drawable {
 
     void SetAlpha(const Uint8 alpha);
 
+    void SetColorMod(const SDL_Color color);
```

Comment:
Please use reference 
```suggestion
    void SetColorMod(const SDL_Color& color);
```